admin/src/client.rs

```diff
@@ -130,15 +140,19 @@ impl Client {
             .context("failed to extract json body from delete response")
     }
 
-    async fn get<R: DeserializeOwned>(&self, path: &str) -> Result<R> {
+    async fn get_raw(&self, path: &str) -> Result<Bytes> {
```

Comment:
Hmm, this can just switch to that service client we made recently right?

Reply:
Yeah I will rewrite admin to re-use the rest of the codebase instead of being its own thing. But not now 🙈